docs/gatsby-config.js

```diff
@@ -103,6 +104,7 @@ const themeOptions = {
               'analyze/registry-images',
               'analyze/weave-report',
               'analyze/sysctl',
+              'analyze/certificates',
```

Comment:
Nit pick: I suggest moving this to line 81 to have the TOC sorted. In the page, the analyser is at the end of the list which is a bit unintuitive. Not all collectors/analysers are sorted, so if you have some bandwidth to address that, I'd appreciate :)